packages/api/src/models/Invoices.ts

```diff
@@ -65,60 +65,60 @@ enum InvoiceV3Status {
 }
 
 export default interface Invoice {
-	id?: string
+	id?: number
 	accountId?: string
 	accountingSystemId?: string
 	address?: string
 	amount?: Money
 	autoBill?: boolean
-	autobillStatus?: Nullable<AutoBillStatus>
-	basecampId?: string
+	autobillStatus?: Nullable<string>
+	basecampId?: number
 	city?: string
 	code?: string
 	country?: string
 	createDate: Date
 	createdAt?: Date
 	currencyCode?: string
 	currentOrganization?: string
-	customerId: string
+	customerId: number
 	datePaid?: Nullable<Date>
-	depositAmount?: Nullable<number>
+	depositAmount?: Nullable<number> // This is an object in the API
 	depositPercentage?: Nullable<string>
 	depositStatus?: DepositStatus
 	description?: string
 	discountDescription?: Nullable<string>
 	discountTotal?: Money
-	discountValue?: number
+	discountValue?: string
 	displayStatus?: DisplayStatus
 	disputeStatus?: Nullable<string>
 	dueDate?: Date
 	dueOffsetDays?: number
 	email?: string
-	estimateId?: string
-	extArchive?: string
+	estimateId?: number
+	extArchive?: number
 	fName?: string
 	fulfillmentDate?: Nullable<Date>
 	generationDate?: Nullable<Date>
 	gmail?: boolean
 	invoiceNumber?: string
-	invoiceId?: string
+	invoiceId?: number
 	language?: string
 	lastOrderStatus?: Nullable<string>
 	lines?: Line[]
 	lName?: string
 	notes?: string
 	organization?: string
 	outstanding?: Money
-	owner?: Owner
-	ownerId?: string
+	owner?: Owner // Not in API - need t o check again.
```

Comment:
This is returned by `GET /invoices/invoices/<id>` with the include `&include[]=owner`
It returns an object with owner details:

```
userid - INT
fname - STRING
lname - STRING
organization - STRING
email - STRING
```